crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -1140,6 +1140,47 @@ def prepare_all_compute_nodes
         end
       end
 
+      # Restart remote resources at target node from cluster node
+      # This is needed for services (like nova-compute) managed by pacemaker.
+      def restart_remote_resources(controller, node)
+        hostname = node[:hostname]
+        out = controller.run_ssh_cmd(
+          "crm --wait node standby remote-#{hostname}",
```

Comment:
Is there a problem with `crm resource restart`  to restart a service? or for remote this is the only way to do this?